### DISCUSSION TITLE
Conversion to RHEL9.

### DISCUSSION THREAD

**Discussion by arizonagroovejet** [oamg/convert2rhel#859] [General]
Is there a timeline for this tool supporting conversions from Rocky/Alma/Oracle 9 to RHEL 9?

Given Red Hat promote this as a supported tool, I'm surprised they didn't ensure support for converting to RHEL 9 before the recent action taken against rebuilder distros.

**Comment by scoobaspeaz** [comment]
I too am looking for this info. I was surprised there was not a way yet.

**Comment by bocekm** [comment]
Hi, we want to allow that in the future for sure. We don't have a clear plan for when we'll deliver that though. Most likely not in the next 3 months.

**Comment by arizonagroovejet** [comment]
@bocekm Why isn't adding support for RHEL 9 a priority for Red Hat? (I ask you direct because I believe you work for Red Hat.) Surely it makes good sense business wise for Red Hat to do everything it can to make switching from one of the rebuild distros it's recently attempted to kill off, to paying for RHEL, as easy as possible for people.

Imagine someone looking at hundreds of AlmaLinux 9 servers. They've explained the situation to management which is willing to start paying for RHEL 9 to minimise disruption in the case of the need or decision to switch distros. They see they can't use convert2RHEL to avoid replacing all their existing servers. So maybe they think, we don't have to run RHEL or a clone, if we have to replace all our servers anyway even when switching to RHEL, why start giving money to Red Hat, the company that has forced us to change distros, which also killed CentOS 8 eight years early just before we were about to starting migrating to it, and can't be bothered to update it's conversion tool to help people switch to RHEL.

**Reply by bocekm** [reply]
Converting Rocky or Alma to RHEL 8 is possible with upstream convert2rhel, but that's not what Red Hat currently supports. Our team's velocity is limited and we have higher priorities than conversions of Rocky/Alma 8/9 to RHEL 8/9 now.

**Reply by bocekm** [reply]
Opening a support ticket or speaking with sales to demonstrate demand will help prioritize the work on conversions to RHEL 9.
